blog/css/main.css.pp

```diff
@@ -161,33 +161,32 @@
   width: 50%;
 }
 
-pre, div.pygments {
+pre, div.pygments, .SCodeFlow {
```

Comment:
The changes in this file are progress at getting code to look right in Scribble. Happy to remove this work-in-progress from the PR if you want.